### ISSUE TITLE
onChange

### ISSUE THREAD

**Issue by AppWerft** [dbankier/TiDialogs#20]
in README onChange in source onchange ;-)

**Comment by dbankier** [comment]
I believe that was your change :wink:

**Comment by AppWerft** [comment]
You are right, I hj´ave a new version. onChange delivered the same data as the click event. This is nice for live updating of model. Should I PR?
